.changeset/fifty-flies-count.md

```diff
@@ -0,0 +1,11 @@
+---
+'@xstate/react': major
+---
+
+The Context object returned from `createActorContext(...)` now provides a `.useSnapshot()` hook, which replaces `useActor()`:
+
+```ts
+// instead of this:
+// const [state, send] = SomeContext.useActor();
+const [state, send] = SomeContext.useSnapshot();
```

Comment:
what do you think about exporting the same thing~ from the root of the package as well?
```ts
import { useSnapshot } from '@xstate/react'

const actorRef = useActorRef(machine)
const [snapshot, send] = useSnapshot(actorRef)
```

Reply:
like, i'd probably never use it... but then things put on the actor context would match better what we export from `@xstate/react`

Reply:
Sure, good symmetry.